report_templates/entitlements.erb

```diff
@@ -0,0 +1,41 @@
+<%#
+name: Entitlements
+snippet: false
+model: ReportTemplate
+require:
+- plugin: katello
+  version: 3.14.0
+%>
+
```

Comment:
I think this would cause extra empty line, ideally also add - to the comment closing erb tag